firefox-ios/Client/TabManagement/TabWebViewPreview.swift

```diff
@@ -41,7 +41,7 @@ final class TabWebViewPreview: UIView, ThemeApplicable {
 
     // MARK: - Layout
     private func setupLayout() {
-        addSubviews(topStackView, webPageScreenshotImageView, addressBarBorderView, bottomStackView)
+        addSubviews(webPageScreenshotImageView, addressBarBorderView, topStackView, bottomStackView)
```

Comment:
why did we need to move this?

Reply:
The stack view was not visible, so I moved it some layers up in the hierarchy.